packages/desktop-client/package.json

```diff
@@ -58,10 +58,10 @@
   },
   "scripts": {
     "start": "cross-env PORT=3001 react-app-rewired start",
-    "start:browser": "cross-env ./bin/watch-browser",
+    "start:browser": "cross-env ./bin/watch-browser.js",
     "watch": "cross-env BROWSER=none yarn start",
     "build": "cross-env INLINE_RUNTIME_CHUNK=false react-app-rewired build",
-    "build:browser": "cross-env ./bin/build-browser",
+    "build:browser": "cross-env ./bin/build-browser.js",
```

Comment:
Since this doesn’t define any env vars or run a shell script can these both be changed to e.g. `node ./bin/build-browser.js`?